server/views/outcomes/casesInProgress.njk

```diff
@@ -1,3 +1,76 @@
+  {% endif %}
+{% endblock %}
+
+{% set tableData = {
+  attributes: {
+    'data-module': 'moj-sortable-table'
+  },
+  head:
+  [
+    { text: "Defendant" },
+    { text: "Outcome type" },
+    { text: "Probation status" },
+    { text: "Offence", "classes": "govuk-!-width-one-third" },
+    { text: ('
+      <button id="button-hearing-outcome-sort" type="button" data-sort="' + params.hearingDateSort + '"
+          aria-controls="button-hearing-outcome-sort" class="pac-sort-button">Hearing date
+      </button>
+    ') | safe,
+      format: 'numeric',
+      attributes: {
+      "aria-sort": params.hearingDateSort
+    }
+    },
+    { text: "Action" }
+  ],
+  rows: []
+} %}
+
+{% for item in data %}
+
+  {% set offences = [] %}
+
+  {% set notMatched = item.probationStatus | string | lower == "possible ndelius record" %}
+  {% if item.offences | length === 1 %}
+    {% for offence in item.offences %}
+      {{ offences.push(offence | escape) }}
+    {% endfor %}
+    {% elif item.offences | length %}
+    {{ offences.push('<ol class="govuk-list govuk-list--number govuk-!-margin-bottom-0">') }}
+    {% for offence in item.offences %}
+      {{ offences.push('<li>' + offence | escape + '</li>') }}
+    {% endfor %}
+    {{ offences.push('</ol>') }}
+  {% endif %}
+
+
+  {% set defendantFullName = (item.name.forename1 + ' ' + item.name.surname) if (item.name.forename1 and item.name.surname) else item.defendantName %}
+  {% set a11yTitle = 'View case for defendant ' + defendantFullName | escape | apostropheInName | properCase | removeTitle %}
+  {% set crnDisplay = '<div class="pac-secondary-text govuk-body-s govuk-!-margin-top-1">' + item.crn if item.crn else '' + '</div>' %}
+  {%- set tableRow = [
+    { text: ('<a href="/' + params.courtCode + '/hearing/' + item.hearingId + '/defendant/' + item.defendantId + '/summary" class="pac-defendant-link govuk-!-font-weight-bold govuk-link govuk-link--no-visited-state" aria-label="' + a11yTitle + '">' + defendantFullName | escape | apostropheInName | properCase | removeTitle + '</a>' + crnDisplay ) | safe },
+    { text: item.hearingOutcomeDescription | safe },
+    { text: ('<div>' + ('<span class="moj-badge moj-badge--red pac-badge">Possible NDelius Record</span>' | safe if notMatched ) + ('<span class="moj-badge moj-badge--black pac-badge">Psr</span>' | safe if item.awaitingPsr) + ('<span class="moj-badge moj-badge--black pac-badge">Breach</span>' | safe if item.breach) + ('<span class="moj-badge moj-badge--black pac-badge">Sso</span>' | safe if item.suspendedSentenceOrder) + '</div>'+ (item.probationStatus | string | capitalize if not notMatched) + ('<span data-cy="previously-known-termination-date" class="govuk-caption-m">Order ended ' + moment(item.previouslyKnownTerminationDate, 'YYYY-MM-DD').format("D MMMM YYYY") | escape + '</span>' if item.previouslyKnownTerminationDate and item.probationStatus == 'Previously known')) | safe },
```

Comment:
The ticket doesn't state to display breach, previously known termination date, Psr, Sso. Please remove them. You wouldn't get this data from the API anyway